packages/edit-site/src/components/page-pages/index.js

```diff
@@ -109,16 +122,65 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				renderFilter: () => {
```

Comment:
I wanted to have something working, so we can iterate, though I believe this could become declarative. Something along these lines:

```js
{
  filter: {
    type: 'enum', // List of items
    items: authors,
    onSelect: ( view ) => {}
  }
}
```